### DISCUSSION TITLE
Can someone explain to me what "Output channels" are?

### DISCUSSION THREAD

**Discussion by lucasbs97** [openclimatefix/metnet#60] [General]
I am having trouble figuring out what Output channels mean and how to prepare the target data. From what I see in the README example, the target data has 5 dimensions:

1º: Batch size
2º: Forecast steps to train the model on
3º: Output channels
4º: Heigh of the output image
5º: Width of the output image

Doesn't the model make the prediction for the center of the input data per lead time per sample? If so, what does the parameter Output channels mean?

Thanks in advance.

**Comment by jacobbieker** [comment]
Output channels is how many "variables" or targets of interest that you want the model to predict for, or the number of bins you want for the probablistic output. For example, in the original MetNet-1/-2 this was just the precipitation rate, divided into 512 bins, so the output channels would be 512. For some of the work we use it for, its up to 11 satellite imagery channels, so would be 11 for that, if that makes sense?

**Reply by lucasbs97** [reply]
I think I got it now, thank you!
